CONTRIBUTING.md

```diff
@@ -61,12 +57,28 @@ However, when speaking about those or explaining we use `actor` noun form: `stor
 * Documentation
 * Code comments
 
-## Pull Request Process
+## Development
+
+Following section explains various suggestions and procedures while development of Thanos.
+
+### Prerequisites
+
+* It is strongly recommended that you use OSX or popular Linux distributions systems e.g. Ubuntu, Redhat, or OpenSUSE for development.
```

Comment:
Why is it strongly recommended to use a `popular` linux distro vs something like Nix or Gentoo?

Reply:
s/Redhat/Red Hat/